data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/remedial_action/range_action/PstRange.java

```diff
@@ -30,8 +36,60 @@
      * @param networkElement: PST element to modify
      */
     @JsonCreator
-    public PstRange(@JsonProperty("networkElement") NetworkElement networkElement) {
-        super(networkElement);
+    public PstRange(@JsonProperty("id") String id,
+                    @JsonProperty("name") String name,
+                    @JsonProperty("operator") String operator,
+                    @JsonProperty("usageRules") List<UsageRule> usageRules,
+                    @JsonProperty("ranges") List<Range> ranges,
+                    @JsonProperty("networkElement") NetworkElement networkElement) {
+        super(id, name, operator, usageRules, ranges, networkElement);
+    }
+
+    public PstRange(String id,
+                    NetworkElement networkElement) {
+        super(id, networkElement);
+    }
+
+    @Override
+    public void synchronize(Network network) {
+        TwoWindingsTransformer transformer = network.getTwoWindingsTransformer(networkElement.getId());
+        PhaseTapChanger phaseTapChanger = transformer.getPhaseTapChanger();
+        int highTapPosition = phaseTapChanger.getHighTapPosition();
+        int lowTapPosition = phaseTapChanger.getLowTapPosition();
+        RangeDefinition rangeDefinition;
+        if (lowTapPosition <= 0) {
+            rangeDefinition = RangeDefinition.CENTERED_ON_ZERO;
+        } else {
+            rangeDefinition = RangeDefinition.STARTS_AT_ONE;
+        }
+        Range absoluteFixedRange = new Range(lowTapPosition, highTapPosition, RangeType.ABSOLUTE_FIXED, rangeDefinition);
+        addRange(absoluteFixedRange);
```

Comment:
Each time we synchronize the CRAC we add a Range, which is never deleted.

Reply:
yes, indeed I will synchronize the low tap of the PST, it will be possible to delete it in the desynchronize method.